src/components/ChannelDetails.js

```diff
@@ -10,7 +10,7 @@ import { useNavigate } from 'react-router-dom';
 import { MdRemoveCircleOutline } from 'react-icons/md';
 
 // Internal Compoonents
```

Comment:
In this file, Button is used from sharedstyling. Change it to ButtonV2. See at line no. 610.